paymentintent.go

```diff
@@ -37,21 +37,21 @@ type PaymentIntentNextActionType string
 
 // List of values that PaymentIntentNextActionType can take.
 const (
-	PaymentIntentNextActionAuthorizeWithURL PaymentIntentNextActionType = "authorize_with_url"
+	PaymentIntentNextActionTypeRedirectToURL PaymentIntentNextActionType = "redirect_to_url"
 )
 
 // PaymentIntentStatus is the list of allowed values for the payment intent's status.
 type PaymentIntentStatus string
 
 // List of values that PaymentIntentStatus can take.
 const (
-	PaymentIntentStatusCanceled             PaymentIntentStatus = "canceled"
-	PaymentIntentStatusProcessing           PaymentIntentStatus = "processing"
-	PaymentIntentStatusRequiresCapture      PaymentIntentStatus = "requires_capture"
-	PaymentIntentStatusRequiresConfirmation PaymentIntentStatus = "requires_confirmation"
-	PaymentIntentStatusRequiresSource       PaymentIntentStatus = "requires_source"
-	PaymentIntentStatusRequiresSourceAction PaymentIntentStatus = "requires_source_action"
-	PaymentIntentStatusSucceeded            PaymentIntentStatus = "succeeded"
+	PaymentIntentStatusCanceled              PaymentIntentStatus = "canceled"
+	PaymentIntentStatusProcessing            PaymentIntentStatus = "processing"
+	PaymentIntentStatusRequiresCapture       PaymentIntentStatus = "requires_capture"
+	PaymentIntentStatusRequiresConfirmation  PaymentIntentStatus = "requires_confirmation"
+	PaymentIntentStatusRequiresAction        PaymentIntentStatus = "requires_action"
+	PaymentIntentStatusRequiresPaymentMethod PaymentIntentStatus = "requires_payment_method"
```

Comment:
You may have done this on purpose to make the diff easier to read (if so, thanks!, but we should re-alphabetize the list now that the names changed.

Reply:
Damn alphabetization. Nice catch, fixed!